authorize/grpc.go

```diff
@@ -33,11 +33,8 @@ func (a *Authorize) Check(ctx context.Context, in *envoy_service_auth_v3.CheckRe
 
 	querier := storage.NewTracingQuerier(
```

Comment:
we don't seem to use .Traces() of this querier outside of tests, do we need it here?

Reply:
Tracing was probably removed with audit logging. It was a product feature. I guess I can also remove it.